Update MosaikHeer.scn

ghukar opened the pull request:
Main changes:

- IEMs: Vocal Stereo-IEM-Bus split into two Mono-IEM Busses to provide IEMs for up to 6 musicians
- Drums: reorganized channels (added "snare bottom" channel, only one e-drum channel, removed third tom channel)
- Drums: added "DRM SMCK"-Bus for parallel compressing drums (deactivated by default)
- Bass: added channel for "Synth Bass"
- Keys: extension to 4 channels (2 linked channels for main piano sound, 2 separate channels e.g. for pads and arps)
- Vocals: added "Leisure Compressor" as insert for vocals 1 and 2
- FX: replaced "ACU REV" with "ROOM REV" (room reverb dedicated for live stream only) to give better live stream mix
- FX: changed Delay for a Delay without Reverb
- FX: DCA 8 controls send to Delay
- Live Stream: added "Stereo Combinator" as insert to provide multiband compression

To use parallel compression on drums ("DRM SMCK" Bus): adjust the input gain of FX6 (Ultimo Compressor) during loud part of a song so that the Gain Reduction meter shows heavy compression (e.g. -10). Blend in volume of DRM SMCK Bus to taste.

To use Leisure Compressor on Vocals 1 and 2: adjust Peak Reduction during loud part of a song so that Gain Reduction meter shows approx. -6dB on peaks.

ghukar committed:
Update MosaikHeer.scn

Main changes:
- IEMs: Vocal Stereo-IEM-Bus split into two Mono-IEM Busses to provide IEMs for up to 6 musicians
- Drums: reorganized channels (added "snare bottom" channel, only one e-drum channel, removed third tom channel)
- Drums: added "DRM SMCK"-Bus for parallel compressing drums (deactivated by default)
- Bass: added channel for "Synth Bass"
- Keys: extension to 4 channels (2 linked channels for main piano sound, 2 separate channels e.g. for pads and arps)
- Vocals: added "Leisure Compressor" as insert for vocals 1 and 2
- FX: replaced "ACU REV" with "ROOM REV" (room reverb dedicated for live stream only) to give better live stream mix
- FX: changed Delay for a Delay without Reverb
- FX: DCA 8 controls send to Delay
- Live Stream: added "Stereo Combinator" as insert to provide multiband compression

To use parallel compression on drums ("DRM SMCK" Bus): adjust the input gain of FX6 (Ultimo Compressor) during loud part of a song so that the Gain Reduction meter shows heavy compression (e.g. -10). Blend in volume of DRM SMCK Bus to taste.

To use Leisure Compressor on Vocals 1 and 2: adjust Peak Reduction during loud part of a song so that Gain Reduction meter shows approx. -6dB on peaks.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -12,15 +12,15 @@
+/config/routing REC
 /config/userctrl/A WH
-/config/userctrl/A/enc "X001" "X101" "X209" "X202"
+/config/userctrl/A/enc "X001" "X101" "X203" "X205"
```
Can you explain, what has changed here?

ghukar replied:
This line describes the assignment of the rotary encoders in the ASSIGN section to the FX parameters. Since the delay (in slot 3: "X2") was changed the parameter number changed, too (from parameter 09 to 03 and from 02 to 05 / Feedback and balance between delay/chorus).

(see page 104 here: https://wiki.munichmakerlab.de/images/1/17/UNOFFICIAL_X32_OSC_REMOTE_PROTOCOL_%281%29.pdf)

cabcookie replied:
That makes sense. Thank you

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -35,189 +35,189 @@
 /ch/01/gate OFF GATE -56.0 42.0 1 5.02  25 0
 /ch/01/gate/filter ON 3.0 85.3
-/ch/01/dyn ON COMP PEAK LOG -22.0 5.0 2 4.00 63 0.02  55 POST 0 100 OFF
+/ch/01/dyn ON COMP PEAK LOG -13.0 10 0 4.00 66 0.02  55 POST 0 75 OFF
```
Do you think, we should keep the values for dynamics and EQs as in the previous version, or should we take the current one? Why did those changes happen? What do you think was the reason for the change? Did we change the acoustic setup of the drums?

ghukar replied:
We can keep the previous values. Since there are a lot of variables in play when it comes to drums (heavy vs. light hitter, choice of drum sticks, dampening etc., for compressor shown here also gain setting of channel) those would have to be adjusted every Sunday anyway.

Regarding the setup of the drums: I find myself using the drum tortillas on snare only, even with heavy hitters. But previous settings should work for this, too.

cabcookie replied:
Okay, I will roll back to the settings of version 4.4.0 after we had the drum calibration session.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -278,148 +278,148 @@
 /ch/11/grp %00000010 %000000
 /ch/11/automix OFF  +0.0
-/ch/12/config "E Git" 1 GN 12
+/ch/12/config "Synth Bass" 1 GN 12
```
Can we call this misc and use default settings (EQ neutralized as well as Dynamics)? You are the only one with the Synth Bass and you might not play it that often, but we will have one additional instrument from time to time. Saving the setting for the synth bass as a preset might do the job.

cabcookie replied:
Forget it. I think we just have a Synth Bass and in the rare cases we have another instrument, we just use this or another channel for it.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -547,14 +547,14 @@
+/ch/17/dyn OFF COMP PEAK LOG -20.5 3.0 4 3.50 22 0.02  37 POST 0 100 OFF
 /ch/17/dyn/filter ON 2.0 1k97
-/ch/17/insert OFF POST OFF
+/ch/17/insert ON POST FX5L
```
I am not a big fan of this compressor. However, I would try to play around with it again to get more familiar with it. Thus, can we set the standard to OFF? Everyone can enable it if they want. Can you make sure, you add this to the PR description?

ghukar replied:
Okay, agreed. I'll turn it off by default and the channel compressor on again.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -564,29 +564,29 @@
+/ch/18/dyn OFF COMP PEAK LOG -18.5 3.0 4 2.00 10 0.02  40 POST 0 100 OFF
 /ch/18/dyn/filter ON 1.0 1k97
-/ch/18/insert OFF POST OFF
+/ch/18/insert ON POST FX5R
```
Same here please.

ghukar replied:
Done.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -926,7 +926,7 @@
 /ch/28/grp %00000000 %000000
 /ch/28/automix OFF  +0.0
-/ch/29/config "Ambnt L" 1 YEi 3
+/ch/29/config "Ambnt L" 1 YEi 5
```
Are you using the ambient channels? I am not anymore so I believe, we can get rid of them. What do you think?

ghukar replied:
Agreed and changed.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -1254,50 +1254,50 @@
 /auxin/08/mix/16 ON   -oo
 /auxin/08/grp %00001000 %000000
-/fxrtn/01/config "acu rev" 1 MG
+/fxrtn/01/config "room rev" 1 MG
```
Great idea. Thank you!!!

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -1598,7 +1598,7 @@
 /bus/08/mix/06 ON   -oo
 /bus/08/grp %00000000 %000000
-/bus/09/config "Vox L" 1 CY
+/bus/09/config "Vox 1" 1 CY
```
Thank you. Great

cabcookie reviewed (commented):
Overall, great changes. Thanks a lot for your work. I added a couple of notes. Would be great, if you can incorporate some of the requested changes or reply to the comments.

cabcookie commented on scenes/MosaikHeer.scn:
```diff
@@ -1,8 +1,8 @@
-#2.7# "MosaikHeer_v4" "" %000000000 1                                                                                                 
-/config/chlink OFF ON ON OFF OFF OFF ON OFF OFF OFF OFF OFF ON OFF ON OFF
+#2.7# "MosaikHeer_15-03" "" %000000000 1                                                                                       
```
Can you name it MosaikHeer_V5?

cabcookie replied:
Did this change already

cabcookie committed:
feat: moving version number to AUX1

cabcookie committed:
chore(release): 5.0.0

cabcookie committed:
feat: added changes to CHANGELOG

cabcookie committed:
chore(release): 5.1.0

cabcookie commented:
I changed the name of DCA8 and bumped the version already. Let's discuss the other open points, before we move this into the main branch

cabcookie committed:
feat: adding the patch field

cabcookie committed:
chore(release): 5.2.0

ghukar committed:
Update MosaikHeer.scn



Main changes:

    IEMs: Vocal Stereo-IEM-Bus split into two Mono-IEM Busses to provide IEMs for up to 6 musicians
    Drums: reorganized channels (added "snare bottom" channel, only one e-drum channel, removed third tom channel)
    Drums: added "DRM SMCK"-Bus for parallel compressing drums (deactivated by default)
    Bass: added channel for "Synth Bass"
    Keys: extension to 4 channels (2 linked channels for main piano sound, 2 separate channels e.g. for pads and arps)
    Vocals: added "Leisure Compressor" as insert for vocals 1 and 2 (default: turned off)
    FX: replaced "ACU REV" with "ROOM REV" (room reverb dedicated for live stream only) to give better live stream mix
    FX: changed Delay for a Delay without Reverb
    FX: DCA 8 controls send to Delay
    Live Stream: added "Stereo Combinator" as insert to provide multiband compression

To use parallel compression on drums ("DRM SMCK" Bus): adjust the input gain of FX6 (Ultimo Compressor) during loud part of a song so that the Gain Reduction meter shows heavy compression (e.g. -10). Blend in volume of DRM SMCK Bus to taste.

To use Leisure Compressor on Vocals 1 and 2: turn the effect insert ON. Then adjust Peak Reduction during loud part of a song so that Gain Reduction meter shows approx. -6dB on peaks.

To set up the Stereo Combinator for live stream: Set Threshold during loud part so that gain reduction is approx. 3dB.

cabcookie committed:
fix: redundance

cabcookie committed:
fix: rollback drum settings to v4.5.0

cabcookie commented:
Wonderful changes. Thanks a lot @ghukar !!!

cabcookie committed:
chore(release): 5.2.1
